tools/GenerateNewControlProject.ps1

```diff
@@ -107,4 +107,75 @@ $xml.Save($testAppProject)
 FindAndReplaceInFile ($muxControlsDir + "\dev\Telemetry\RuntimeProfiler.h") "(\s*ProfId_Size.*\s*})" @"
 
         ProfId_$controlName,`$1
-"@
\ No newline at end of file
+"@
+# Randomize class name for multiple use of class in one powershell
+$id = get-random
+
+# We need double backslash for C# strings below
+$cleanMuxControlsDir = $muxControlsDir.Replace("\","\\") + "\\"
+
+echo "$cleanMuxControlsDir"
+
+$assemblies=(
+	"System","EnvDTE","EnvDTE80"
+)
+
+$source="
```

Comment:
You can use [here-strings](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-powershell-1.0/ee692792(v=technet.10)?redirectedfrom=MSDN) in PowerShell to avoid needing to escape every internal quote mark:

```
$str = @"
This contains "multiple" "quote marks" but does not require escaping.
"@
```

Reply:
For some reason, at some point that did not work correctly, that's why I solved it that way. But now it works, so yeah. Thanks for pointing that out!